src/inputFile_impl.h

```diff
@@ -392,11 +393,25 @@ void InputFile<t_System>::check_lammps_command( std::string line,
     if ( keyword.compare( "neigh_modify" ) == 0 )
     {
         known = true;
-        for ( std::size_t i = 0; i < words.size(); i++ )
+        std::size_t i = 1;
+        while ( i < words.size() )
+        {
             if ( words.at( i ).compare( "every" ) == 0 )
             {
                 comm_exchange_rate = std::stoi( words.at( i + 1 ) );
+                i += 2;
+            }
+            else if ( words.at( i ).compare( "one" ) == 0 )
```

Comment:
Why did you chose to call the keyword "one"?

Reply:
That's the equivalent lammps option.